### PR TITLE
SUP-32950 Adding ignore MRP reset on metadata update - configuration …

### PR DESCRIPTION
**Pull request by bw-kaltura**
…based

### CODE REVIEW

File: plugins/scheduled_task/lib/MediaRepurposingHandler.php [hunk 1]
```diff
@@ -6,6 +6,8 @@
  */
 class MediaRepurposingHandler implements kObjectDataChangedEventConsumer
 {
+    	const IGNORE_MRP_RESET_ON_METADATA_UPDATE = "ignoreMRPResetOnMetadataUpdate";
```
Comment (inbal-ben-david) [inline plugins/scheduled_task/lib/MediaRepurposingHandler.php:9]: change to something like:
[excluded_MRP_ids_from_metadata_update]

0 = MRP ID1
1 = MRP ID2

and remove MR_PROFILES const

File: plugins/scheduled_task/lib/MediaRepurposingHandler.php [hunk 2]
```diff
@@ -70,19 +72,29 @@ public function shouldConsumeDataChangedEvent(BaseObject $object, $previousVersi
...
-			$items = $mediaRepurposingProfile->getObjectFilter()->getAdvancedSearch()->getItems(); // always have items in advance search because that how the MR is build
-			foreach($items as $item) {
-				if ($item instanceof MetadataSearchFilter && $item->getMetadataProfileId() == $metadataProfileId)
-					$mediaRepurposingProfilesWithSearchOnGivenMetadataId[] = $mediaRepurposingProfile->getId();
-			}
-		}
-		return $mediaRepurposingProfilesWithSearchOnGivenMetadataId;
+        	$allMediaRepurposingProfilesOnPartner = ScheduledTaskProfilePeer::retrieveBySystemName(MediaRepurposingUtils::MEDIA_REPURPOSING_SYSTEM_NAME, $partnerId);
+        	$mediaRepurposingProfilesWithSearchOnGivenMetadataId = array();
+        	$ignoreMRPResetOnMetadataUpdateConf = kConf::get(self::IGNORE_MRP_RESET_ON_METADATA_UPDATE, kConfMapNames::RUNTIME_CONFIG, array());
```
Comment (inbal-ben-david) [inline plugins/scheduled_task/lib/MediaRepurposingHandler.php:77]: adjust with new name and then you wont need to init the array or explode

File: plugins/scheduled_task/lib/MediaRepurposingHandler.php (hunk 1 above)
Comment (bw-kaltura) [reply plugins/scheduled_task/lib/MediaRepurposingHandler.php]: done

File: plugins/scheduled_task/lib/MediaRepurposingHandler.php (hunk 2 above)
Comment (bw-kaltura) [reply plugins/scheduled_task/lib/MediaRepurposingHandler.php]: done

**Review by inbal-ben-david — APPROVED** [review]

**Pull request merged**